### PR TITLE
v4 upgrade handler

### PR DESCRIPTION
**Pull request by jhernandezb**

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -248,7 +248,139 @@ volumes:
 node:
   runner: integration
 ---
+kind: pipeline
```
Comment (yubrew) [inline .drone.yml:251]: is all of this extra integration tests?

Comment (jhernandezb) [reply .drone.yml:251]: is simulating the on chain upgrade by submitting proposal to gov etc.

File: app/upgrades.go [hunk 2]
```diff
@@ -3,36 +3,19 @@ package app
...
-		consensusParams.Block.MaxGas = 75_000_000 // 75M
-		app.BaseApp.StoreConsensusParams(ctx, consensusParams)
-
-		// wasm params
-		wasmParams := app.WasmKeeper.GetParams(ctx)
-		wasmParams.CodeUploadAccess = wasmtypes.AllowNobody
-		wasmParams.MaxWasmCodeSize = DefaultMaxWasmCodeSize
-		app.WasmKeeper.SetParams(ctx, wasmParams)
-		return newVM, err
+		return app.mm.RunMigrations(ctx, cfg, vm)
```
Comment (shanev) [inline app/upgrades.go:18]: Should we update the block per year while we're at it?

**Review by shanev — APPROVED** [review]

File: app/upgrades.go (hunk 2 above)
Comment (jhernandezb) [reply app/upgrades.go:18]: aa great question but I'd rather see a different proposal

**Review by yubrew — APPROVED** [review]

**Pull request merged**